tvm-disasm/src/main/kotlin/org/ton/disasm/TvmDisassembler.kt

```diff
@@ -361,6 +396,16 @@ data object TvmDisassembler {
         }
     }
 
+    private fun serializeInstListOperand(insts: List<TvmInst>, raw: JsonElement): JsonObject {
+        val list = serializeInstList(insts)
+        return JsonObject(
+            mapOf(
+                "list" to list,
+                "raw" to raw,
```

Comment:
What do you mean with this `raw` parameter?

Reply:
This is a cell that represents this continuation